Registration_Example.ipynb

```diff
@@ -0,0 +1,342 @@
+   "source": [
+    "#### Verifying Registration\n",
+    "We have just registered `pdarray pda1` and `String str1`. Note that `pdarray pda2` and `String str2` have not been registered. We can verify this using `ak.list_registry`, `is_registered`, or `ak.info`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ak.list_registry returns a python list of all registered object names\n",
+    "print(ak.list_registry())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Object.is_registered() returns a boolean indicating the objects registration status\n",
+    "print(f'pda1 is registered: {pda1.is_registered()}')\n",
+    "print(f'pda2 is registered: {pda2.is_registered()}')\n",
+    "print(f'str1 is registered: {str1.is_registered()}')\n",
+    "print(f'str2 is registered: {str2.is_registered()}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ak.info returns all attributes of an object\n",
+    "# ak.info can be called with a single object, all registered objects, or all objects\n",
+    "\n",
+    "print(\"ak.info('registered_object_name'):\")\n",
+    "print(ak.info('pda1'))\n",
+    "\n",
+    "print('ak.info(ak.RegisteredSymbols):')\n",
+    "print(ak.info(ak.RegisteredSymbols))\n",
+    "\n",
+    "print('ak.info(ak.AllSymbols):')\n",
+    "print(ak.info(ak.AllSymbols))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see `ak.info(ak.RegisteredSymbols)` only contains references to objects related to `pda1` and `str1`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`ak.clear()` removes all non-registered objects from the symbol table, so `ak.info(ak.AllSymbols)` is different after a clear"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Before clear:')\n",
+    "print(ak.info(ak.AllSymbols))\n",
+    "ak.clear()\n",
+    "print('After clear:')\n",
+    "print(ak.info(ak.AllSymbols))\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Disconnecting from Arkouda Server and Attaching Registered Objects\n",
+    "Users can `attach` to objects registered with the server after a disconnect. This enables access to registered objects even if the original python object is lost  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are going to simulate the python client dying. To do this, we set references for `pda1` and `str1` to `None` and disconnect from the server"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pda1 = None\n",
+    "str1 = None"
```

Comment:
I think setting the refs to none will cause the objects to go out of scope and trigger deletions on the Arkouda Server.  It's probably best to remove the Nones and just go with the disconnect.

Reply:
Update:  To answer the question of whether arkouda_server will block a deletion in the event an object is registered, the answer is no  see [deleteMsg](https://github.com/mhmerrill/arkouda/blob/fe4744ce13f0d19f007336ad1de9e9dca58f7153/src/MsgProcessing.chpl#L83)

Reply:
I chatted with @glitch about this a bit and it seems that the delete is only being triggered if an object is attached to and then set to `None` (regardless of registration status)

@glitch can correct me if I'm wrong, but I think we decided that what the notebook is doing now is fine